src/table/table-col-resizable.directive.ts

```diff
@@ -0,0 +1,89 @@
+import { AfterViewInit, Directive, inject, Input } from '@angular/core';
+import { takeUntil } from 'rxjs';
+
+import { ResizableDirective } from '../resizable';
+import { handlePixel } from '../utils';
+
+import {
+  tableBem,
+  TableColumnDefDirective,
+  TableComponent,
+  TableScrollWrapperDirective,
+} from './index';
+
+let tableResizableDynamicStyleUid = 0;
+
+@Directive({
+  selector: '[auiTableColResizable]',
+  standalone: true,
+})
+export class TableColResizableDirective
+  extends ResizableDirective
+  implements AfterViewInit
+{
+  @Input()
+  override minWidth = '60px';
+
+  override readonly manualAdjustment = true;
+
+  private readonly tableColumnDefDirective = inject(TableColumnDefDirective);
+  private readonly tableComponent = inject(TableComponent);
+  private readonly tableScrollWrapperDirective = inject(
+    TableScrollWrapperDirective,
+    {
+      optional: true,
+    },
+  );
+
+  private styleEl: HTMLStyleElement;
+  private sheet: CSSStyleSheet;
+  private hostAttr: string;
+
+  override ngAfterViewInit() {
+    super.ngAfterViewInit();
+
+    this.containerElement = (
+      this.tableScrollWrapperDirective || this.tableComponent
+    ).el.nativeElement;
+
+    this.resizeEnd.pipe(takeUntil(this.destroy$$)).subscribe(width => {
+      const className = tableBem.element(
+        `column-${this.tableColumnDefDirective.cssClassFriendlyName}`,
+      );
+      if (!this.hostAttr) {
+        this.hostAttr = `table-resizable-dynamic-style-${tableResizableDynamicStyleUid++}`;
+        this.containerElement.setAttribute(this.hostAttr, '');
+      }
+      const styleString = `
+       [${this.hostAttr}] .${className} {
+        width: ${handlePixel(width)} !important;
+        flex: none !important;
+      }
+      `;
+      if (CSSStyleSheet) {
```

Comment:
```suggestion
      if (typeof CSSStyleSheet !== 'undefined') {
```

原来的写法在没有 CSSStyleSheet 的环境会 `ReferenceError: CSSStyleSheet is not defined`.